### ISSUE TITLE
broken links

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy-hub#185]
I will collect here a few broken links. Feel free to modify this issue.

* https://galaxyproject.org/Learn/AdvancedWorkflow/Variables - https://galaxyproject.org/learn/advanced-workflow/variables-edit/ # used in the workflow editor https://github.com/galaxyproject/galaxy/pull/3773
* https://galaxyproject.org/duplex/
* https://galaxyproject.org/galaxy101/ -> https://galaxyproject.org/tutorials/g101/
* https://galaxyproject.org/schema/SchemaSpy/index.html

**Comment by felixeye** [comment]
I can also add the following link [https://galaxyproject.org/admin/config/performance/Purge%20Histories%20and%20Datasets/](https://galaxyproject.org/admin/config/performance/Purge%20Histories%20and%20Datasets/)

**Comment by petewall** [comment]
Also broken: https://galaxyproject.org/learn/share/

**Comment by erasche** [comment]
https://galaxyproject.org/Images/Logos

**Comment by caot** [comment]
broken link: https://galaxyproject.org/admin/config/Upload%20via%20FTP/

**Comment by martenson** [comment]
https://galaxyproject.org/admin/config/nginxProxy/

**Comment by polkhe** [comment]
more broken links:

- https://galaxyproject.org/schema/SchemaSpy/index.html
- https://galaxyproject.org/Admin/Datatypes/Adding%20Datatypes looks like should be https://galaxyproject.org/admin/datatypes/adding-complete-datatypes/ ?
- https://galaxyproject.org/Support/Commercial
- https://galaxyproject.org/galaxy101/fa-columns.png
- https://galaxyproject.org/galaxy101/fa-cog.png
- https://galaxyproject.org/galaxy101/historyDetail.png
- https://galaxyproject.org/galaxy101/multiHistoryView.png
- https://galaxyproject.org/duplex/fromReads.png
- https://galaxyproject.org/duplex/fromDCS.png
- https://galaxyproject.org/galaxy101/fa-refresh.png
- https://galaxyproject.org/duplex/histItem.png

**Comment by martenson** [comment]
http://wiki.g2.bx.psu.edu/Learn/Interval%20Operations

**Comment by erasche** [comment]
(Is there any way we could just grep logs for 404s? Surely that would surface a bigger list of missing URLs without this manual and biased discovery process)

**Comment by bgruening** [comment]
@erasche this one? https://github.com/galaxyproject/galaxy-hub/blob/master/Makefile#L29

**Comment by martenson** [comment]
@bgruening I think @erasche means server logs, the actual hits for 404s

**Comment by erasche** [comment]
yeah, what @martenson said. Surely people are trying to access dead links from google / links in docs / etc. Surely it would be easier just to get that, even with the noise from spiders / etc. Or if you had a comprehensive sitemap.xml (as I'm guessing you did for webmaster tools for SEO), that would be a good starting place too, rather than just waiting for people to find them.

**Comment by martenson** [comment]
@erasche My approach was that we can identify patterns of what link types were not properly converted/redirected and have a generic fix. From the beginning we wanted to have an unbroken migration.

**Comment by erasche** [comment]
Ah, interesting. That makes sense under the assumption that specific classes of documents were not converted. From an outsider perspective, my naïve impression is that there is a problem which runs deeper than link types and perhaps we should be more proactive in identifying and resolving these issues. E.g. logos missing vs specific admin page missing vs training docs missing all seem very different.

However, given that these links are still broken 2 months later, perhaps there should be time spent on fixing it, and it'd have to come from one of y'all since the rest of us don't have access to the original wiki. If you could dump the old wiki's contents somewhere, perhaps the community could help out.

**Comment by martenson** [comment]
I think everybody has access to https://moin.galaxyproject.org/

**Comment by erasche** [comment]
TIL! Awesome.

**Comment by dannon** [comment]
> However, given that these links are still broken 2 months later, perhaps there should be time spent on fixing it

Yes, please!  Go for it.

**Comment by tnabtaf** [comment]
Thank you @bgruening, @hexylena, @felixeye. @caot, @martenson, and @polkhe for reporting (and sometimes fixing!) these broken links.

I believe I have now fixed all remaining broken links reported here with [this commit](https://github.com/galaxyproject/galaxy-hub/commit/4c9693f65b8ac94764c18ab56097bfa095d544df).  I finally got to this because I want to create a `paper-cut` issue in hub for broken links, and I I want it to have a clean history and directions for newcomers.
